pallets/identity-management/Cargo.toml

```
@@ -0,0 +1,40 @@
+[package]
+authors = ['Litentry Dev']
+edition = '2021'
+homepage = 'https://litentry.com'
+name = 'pallet-identity-management'
+repository = 'https://github.com/litentry/litentry-parachain'
+version = '0.1.0'
+
+[dependencies]
+# third-party dependencies
+codec = { package = "parity-scale-codec", version = "3.0.0", default-features = false, features = ["derive", "max-encoded-len"] }
+scale-info = { version = "2.1.1", default-features = false, features = ["derive"] }
+log = { version = "0.4.16", default-features = false }
```

Can we have 
```suggestion
log = { version = "0.4", default-features = false }
```
here? The reason is once I got version conflict as other parts of code depend on `0.4` (which will lead to `0.4.17` in Cargo.lock)